packages/shared/src/components/fields/TextArea.tsx

```diff
@@ -0,0 +1,203 @@
+import classNames from 'classnames';
```

Comment:
I fear it might become a bit complex and hard to maintain, so I decided to have them separated, but I understand your concern I think we should find a way to uniform them as a lot of them looks like copy and pasted.

Reply:
Ah yes ok that makes sense for now, perhaps we can just log it and take it from there :)